iked/libcrux_mlkem768_sha3.c

```diff
@@ -36,7 +36,7 @@
 #define KRML_MUSTINLINE inline
 #define KRML_NOINLINE __attribute__((noinline, unused))
 #define KRML_HOST_EPRINTF(...)
-#define KRML_HOST_EXIT(x) fatal_f("internal error")
+#define KRML_HOST_EXIT(x) exit(-1)
```

Comment:
fatalx()?

Reply:
@mfriedl will change this. I am going to update the PR soon with one more bugfix.